### PR TITLE
Enable publishing extension-browser docs to npm

### PR DESCRIPTION
**Pull request by antross**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [ ] Added/Updated related tests.

## Short description of the change(s)

A few changes to include docs in the `extension-browser` package, allow `extension-browser` to be published to npm (just docs), and switch to using the monorepo release script to bump versions of `extension-browser` and point to where to submit packages to the extension stores.

### CODE REVIEW

**Comment by antross** [comment]
I'll leave it for a separate PR. I'm fine if these go out in separate releases.

File: release/lib/skippers.ts [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+export const skipIfBrowserSame = ({ packages }: Context) => {
+    const pkg = packages && packages.get('extension-browser');
+    const version = pkg && pkg.content.version;
+    const oldVersion = pkg && pkg.oldVersion;
+
+    if (version === oldVersion) {
+        return '';
+    }
+
+    return '';
```
Comment (sarvaje) [inline release/lib/skippers.ts]: you are returning always `return ''` is there any error here?

Comment (antross) [reply release/lib/skippers.ts]: Good catch! The `return` inside the `if` check should have a message stating we're skipping publishing as the browser extension is unchanged.

**Comment by antross** [comment]
@sarvaje I fixed the missing skip message if you want to give this another look.

**Pull request merged**